22-ajax-lab/src/01-swapi.jsx

```diff
@@ -10,19 +10,117 @@
  * allowing a user to expand from a Person to a Film she played in,
  * to another Person from that film and so on.
  */
-
 import React from 'react';
+import $ from 'jquery';
+
+const PersonDao = {};
+
+PersonDao.search = function (id) {
+    return $.getJSON(`http://swapi.co/api/people/${id}`);
```

Comment:
I suggest use native browser [fetch api](https://developer.mozilla.org/en/docs/Web/API/Fetch_API) or [axios](https://www.npmjs.com/package/axios) for fetching. Don't really need jquery here.